docs/templates.md

```diff
@@ -399,7 +399,7 @@ The following methods are defined.
 
 * `.ago() -> String`: Format as relative timestamp.
 * `.format(format: String) -> String`: Format with [the specified strftime-like
-  format string](https://docs.rs/chrono/latest/chrono/format/strftime/).
+  format string](https://docs.rs/jiff/latest/jiff/fmt/strtime/index.html#conversion-specifications).
```

Comment:
Nit: since this is user-visible it should have a deprecation period before changing or at least a warning for the deprecated format

Reply:
The formats are *basically* identical, so I'd be surprised if this actually affected anyone. (Also, see the changelog entry.)

Reply:
The problem is that _almost identical_ interfaces will depend on [Hyrum's Law], so any migration is not that easy. OK. then it should be fine with minor issues.  

[Hyrum's Law]: https://www.hyrumslaw.com/